### PR TITLE
Update fs2-core to 1.0.0-M1

### PR DESCRIPTION
**Pull request by redraw-dawn**

### CODE REVIEW

**Comment by fiadliel** [comment]
Thanks, the dust around cats-effect/FS2 seems to be settling, so I suspect the latest version will be something that we can start working on.

File: src/main/scala/org/lyranthe/fs2_mongodb/MongoClient.scala [hunk 1]
```diff
@@ -5,16 +5,18 @@ import fs2._
...
+  }
+
+  def fromSettings[F[_]](settings: MongoClientSettings)(implicit F: Sync[F]): Stream[F, MongoClient] = {
+    createStream(MongoClients.create(settings))
   }
 
-  def fromSettings[F[_]](settings: MongoClientSettings)(
-      implicit F: Sync[F]): Stream[F, MongoClient] = {
-    Stream.bracket(F.delay(MongoClients.create(settings)))(Stream.emit(_), { client =>
+  private def createStream[F[_]](mongoClient: => MongoClient)(implicit F: Sync[F]): Stream[F, MongoClient] = {
```
Comment (Daenyth) [inline src/main/scala/org/lyranthe/fs2_mongodb/MongoClient.scala:17]: What do you think about exposing a `Resource` instead of a stream?

Comment (fiadliel) [reply src/main/scala/org/lyranthe/fs2_mongodb/MongoClient.scala:17]: I think, yes, it's probably the more idiomatic thing to do at this point (as much as there are idioms with unreleased software).

**Comment by redraw-dawn** [comment]
closing as it is obsolete now :)

**Pull request closed without merging**